spec/validator.js

```diff
@@ -50,4 +50,14 @@ describe('Validator constructor', function() {
   it('should have a check method', function() {
     expect(validator.check).to.be.a.function;
   });
+
+  it('should handle undefined data', function(){
+    validator = new Validator(undefined, { name: 'required' })
+    validator.check();
```

Comment:
I think there should be an assertion here that verifies the validation has failed. What do you think @garygreen?